src/Sulu/Bundle/SecurityBundle/Tests/Unit/Security/AuthenticationHandlerTest.php

```diff
@@ -65,7 +65,7 @@ public function setUp(): void
         $router = $this->prophesize(RouterInterface::class);
         $session = $this->prophesize(Session::class);
         $session->get('_security.admin.target_path')->willReturn('/admin/#target/path');
-        $session->set(Security::AUTHENTICATION_ERROR, $this->exception->reveal())->willReturn(null);
+        $session->set(Security::AUTHENTICATION_ERROR, $this->exception->reveal());
```

Comment:
This just calls the `set` method we need I think something like.

Based on https://github.com/phpspec/prophecy/issues/280

it maybe works if we do `->willReturn()` or if we do `->will(fn() => {})` not 100% sure